docs/auth.md

```diff
@@ -24,6 +24,12 @@ To solve this, before any `PipelineResources` are retrieved, all `pods` execute
 a credential initialization process that accesses each of its secrets and
 aggregates them into their respective files in `$HOME`.
 
+Please note: If the `disable-home-env-overwrite` feature flag has been set to
```

Comment:
So, in the case of `disable-home-env-overwrite` (and by default in the future), do we decide that `/tekton/home` is where we write anything that "would be in `$HOME`" ? (like `.gitconfig`, …) I am +:100: on this as we will document it, and we are sure it's always written at the same place.

Reply:
Interesting idea!  I think this might make sense.  I was considering using a more explicit name like `/tekton/creds` since we'd originally planned for /tekton/home to be removed in future.  But if we see this as reusable then I think it makes sense to leave it as-is.

I'm going to write up a one-page design doc and will present it to the WG on wednesday just to quickly get community feedback on that plan.

Reply:
+:100: on both your comments :wink: I really like `/tekton/creds` and having a variable for it is :heart_eyes_cat:

Reply:
👍 awesome, i've implemented both of these now.